french/dormir/reve-etrange.md

```diff
@@ -1 +1,11 @@
-Vous parvenez à vous rendormir mais faites le plus étrange des rêves, dansant le tango avec un éléphant rose !
\ No newline at end of file
+Vous parvenez à vous rendormir mais faites le plus étrange des rêves, dansant le tango avec un éléphant rose !
+
+Vous vous reveillez ce qui semble etre quelques heures plus tard, de tres bonne humeur mais les pieds meurtris pour une etrange raison.
+
+Que faites-vous ?
+
+[Je cherche une sortie](sortie/chercher-la-sortie.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.